conf/script/build-ha-update

```diff
@@ -83,7 +83,11 @@ reset_all() {
 }
 
 echo 'Exporting Consul KV...'
-$consul_bin/consul kv export > $hare_dir/consul-conf-exported.json
+consul_kv_dump=$hare_dir/consul-kv-dump.json
+if [[ -f $consul_kv_dump ]]; do
+    cat $consul_kv_dump | xz > $consul_kv_dump-$(date -Iseconds).xz
```

Comment:
`date -Iseconds` produces string with colons, so there will be no way to download such a file to a Windows machine:

```sh
date -Iseconds
2020-10-20T08:20:21-0600
```

Are we ok with this behavior?

Reply:
Good point. Let me change it...